src/render/painter.js

```diff
@@ -685,7 +685,9 @@ class Painter {
         this.id = layer.id;
 
         this.gpuTimingStart(layer);
-        draw[layer.type](painter, sourceCache, layer, coords, this.style.placement.variableOffsets, this.options.isInitialLoad);
+        if (!painter.transform.projection.unsupportedLayers || !painter.transform.projection.unsupportedLayers.has(layer.type)) {
+            draw[layer.type](painter, sourceCache, layer, coords, this.style.placement.variableOffsets, this.options.isInitialLoad);
```

Comment:
It would be better to not load the data at all, but since the support for these layers will be added soon I think it's easier just to have this simple workaround.

Reply:
Yes that works for now. Is there some redundancy with the other toggles that we could remove (Mainly thinking of the overlap with `supportsCustomLayers`)

Reply:
Good point, I'll merge those. 👌